### ISSUE TITLE
check-logs does not take into account the uuid part of the index name

### ISSUE THREAD

**Issue by richm** [openshift/origin-aggregated-logging#198]
check-logs.sh gets the list of indices like this:
```
Checking for log entry matches between ES and their sources...
   found 2 index(es) [logging.6ce8df18-4579-11e6-bd53-0e92492f330b test.65e07268-457a-11e6-bd53-0e92492f330b]
```
Then for each log message found in elasticsearch, it greps in the container log file like this:
```
Failure - [0/500] log entries found in /var/log/containers/*_logging.6ce8df18-4579-11e6-bd53-0e92492f330b_*.log
```
The uuid `6ce8df18-4579-11e6-bd53-0e92492f330b` is *not* part of the log filename.

**Comment by richm** [comment]
@ewolinetz ptal - this makes check-logs.sh less than useful and/or a source of false positive errors.

**Comment by ewolinetz** [comment]
Good catch, I'll check this out today

**Comment by ewolinetz** [comment]
fixed as of #199
